queries/eager_load.go

```diff
@@ -5,9 +5,9 @@ import (
 	"reflect"
 	"strings"
 
-	"github.com/pkg/errors"
 	"github.com/lbryio/sqlboiler/boil"
 	"github.com/lbryio/sqlboiler/strmangle"
+	"github.com/pkg/errors"
```

Comment:
why did this get moved?

Reply:
I think it was changed by IDE and sorted on save